### ISSUE TITLE
cgroup control

### ISSUE THREAD

**Issue by freeseacher** [hishamhm/htop#916]
hi! 

tnx for greate software! 
i'am using htop for long time, and my is desktop also works under linux with relativly new kernel. 
Today i launched two bzip process while listening music via browser and my music became laggy. 
After latest kubernetes exprience my first thought was i have to do something with bzip process. 
I changed its prio to -20. that did not help ofcause. than i use ionice to change io priority to "be" and prio 5. that did not help.

than i thought about cgroups and how it can help me with music. and my though was about htop that it can create low priority group for me and place pids to it. 

And here am i here. could you please add cgroup config. 
smthing like
```
 sudo cgcreate -g cpu:/cpulimited
 sudo cgset -r cpu.shares=512 cpulimited
 sudo cgclassify -g cpu:cpulimited 1684
```
would be pretty usefull

**Comment by rubyFeedback** [comment]
This would be limited to linux only right? I think it would be better to keep htop as portable as possible.

**Comment by freeseacher** [comment]
yes. that would be linux only. because there are no such mehanics in other systems. 
so ok. should be closed if favour of portability
